full-backend-tests/src/test/java/org/graylog/plugins/views/QueryValidationResourceIT.java

```diff
@@ -30,6 +31,7 @@
 import static org.graylog.testing.completebackend.Lifecycle.CLASS;
 import static org.graylog.testing.graylognode.NodeContainerConfig.GELF_HTTP_PORT;
 import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.Matchers.not;
 import static org.hamcrest.core.IsEqual.equalTo;
 
 @ContainerMatrixTestsConfiguration(serverLifecycle = CLASS)
```

Comment:
It is completely unrelated to your fix Tomas, but I am just wondering... is validation dependent on MongoDB or ES versions? If not, maybe we should limit the number of tests to just one Mongo and one ES version? It would speed up execution on Jenkins.

Reply:
You are right, configured the default versions. Good idea, thanks!